appinfo/info.xml

```diff
@@ -22,6 +22,10 @@ This extension provides the most simple way to add a second factor to user login
 	<settings>
 		<personal>OCA\TwoFactor_Totp\Settings\PersonalPanel</personal>
 	</settings>
+	<commands>
+		<command>OCA\TwoFactor_Totp\Command\VerifySecretCommand</command>
```

Comment:
maybe a single command with a flag "--set-verified=true" or something would be simpler than two commands ?

Reply:
At first, I planned to implement it as one command. But, the other commands generally implemented as two commands (enable/disable, activate/inactivate). Because of that, I thought it is a convention. I'm gonna refactor it.